package.json

```diff
@@ -64,7 +64,7 @@
     "tslib": "^2.7.0",
     "typedoc": "^0.26.7",
     "typedoc-plugin-markdown": "^4.2.7",
-    "typescript": "~5.6.2",
+    "typescript": "~5.4.5",
```

Comment:
issue: why are you downgrading typescript?

Reply:
Fixed